lib/travis/build/config.rb

```diff
@@ -96,6 +96,7 @@ def sc_data
           ),
         },
         network: {
+          wait?: ENV.fetch('TRAVIS_BUILD_NETWORK_WAIT', 'true').to_bool,
```

Comment:
Are we sure that naming this with a `?` won't cause issues? I think it is much more clear but I don't see any examples of this usage anywhere.

Something to monitor during testing!

Reply:
I checked the gem, and it actually does this for us (https://github.com/svenfuchs/hashr)